node/as/thrift.js

```diff
@@ -65,6 +88,17 @@ function TChannelThriftRequest(options) {
     self.tchannelThrift = options.tchannelThrift;
 }
 
+function health(self, req, head, body, callback) {
+    var status = self.healthCheckCallback();
```

Comment:
Let's add `assert('ok' in status, 'Status must have ok field')` and `assert('message' in status, 'Status must have message field')`